modules/host/host.go

```diff
@@ -4,10 +4,12 @@ import (
 	"errors"
 	"net"
 	"os"
+	"sync"
 
+	"github.com/NebulousLabs/Sia/crypto"
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/modules/consensus"
-	"github.com/NebulousLabs/Sia/sync"
+	safesync "github.com/NebulousLabs/Sia/sync"
```

Comment:
hmm, I don't think so. In my mind, the grouping indicates "distance" from your project.